yahoo-finance/YahooFinance.py

```diff
@@ -0,0 +1,12 @@
+from yahoo_finance import Share
+
+def get_data(ticker, start_date, end_date):
+	"""Creates wrapper instance of yahoo_finance Share class.
```

Comment:
Could you make this more about what the function does than the implementation details?

Something simple like `"Gets stock data from Yahoo Finance between dates"`.